### PR TITLE
feat: add art quiz as an onboarding reward screen

### PR DESCRIPTION
**Pull request by laurabeth**
The type of this PR is: **Feat**

This PR solves [GRO-1252]

### Description

This PR contains the logic that conditionally adds the Art Quiz as an option to the user onboarding flow. The logic is outlined [here](https://www.figma.com/file/F12JFyxE0DR0qCGX2nap5E/Art-Quiz%3A-Onboarding-Phase-2?node-id=36%3A32&t=RXDUidcrzrM8Ffxb-0).

Tracking is implemented and follows the same patterns as previous onboarding reward screen options.

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[GRO-1252]: https://artsyproduct.atlassian.net/browse/GRO-1252?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Components/Onboarding/Hooks/useOnboardingContext.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-relative-import-paths/no-relative-import-paths */
```
Comment (damassi) [inline src/Components/Onboarding/Hooks/useOnboardingContext.tsx:1]: Why add this?

File: src/Components/Onboarding/Views/OnboardingThankYou.tsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-relative-import-paths/no-relative-import-paths */
```
Comment (damassi) [inline src/Components/Onboarding/Views/OnboardingThankYou.tsx:1]: ☝️

File: src/Components/Onboarding/Hooks/useOnboardingContext.tsx (hunk 1 above)
Comment (dzucconi) [reply src/Components/Onboarding/Hooks/useOnboardingContext.tsx:1]: Yeah just autofix the errors

Comment (laurabeth) [reply src/Components/Onboarding/Hooks/useOnboardingContext.tsx:1]: I didn't... I thought you did @dzucconi. 😆 I'll remove.

Comment (laurabeth) [reply src/Components/Onboarding/Hooks/useOnboardingContext.tsx:1]: @damassi liveshare must be haunted. ;)

File: src/Components/Onboarding/Views/OnboardingThankYou.tsx (hunk 2 above)
Comment (laurabeth) [reply src/Components/Onboarding/Views/OnboardingThankYou.tsx:1]: Like... I would own it if I did it, but I did not add this line. I'm guessing that someone's VSCode thought this was an ok autofix option?

File: src/Components/Onboarding/Hooks/useOnboardingContext.tsx (hunk 1 above)
Comment (damassi) [reply src/Components/Onboarding/Hooks/useOnboardingContext.tsx:1]: its all madness 😅

File: src/Components/Onboarding/Views/OnboardingThankYou.tsx (hunk 2 above)
Comment (damassi) [reply src/Components/Onboarding/Views/OnboardingThankYou.tsx:1]: this has happened to me often, where i'll accidentally right click and disable, but beside that just thinking how liveshare works, how complicated it is... who knows?

**Pull request merged**